gorm-tools/src/main/groovy/gorm/tools/repository/bulk/BulkableRepo.groovy

```diff
@@ -82,10 +82,10 @@ trait BulkableRepo<D> {
 
         syncJobArgs.entityClass = getEntityClass()
         SyncJobContext jobContext = syncJobService.createJob(syncJobArgs, dataList)
-        //FIXME why are we setting session: true here? explain. should it be default?
-        def asyncArgs = jobContext.args.asyncArgs.session(true)
+        //XXX why are we setting session: true here? explain. should it be default?
+        //def asyncArgs = jobContext.args.asyncArgs.session(true)
```

Comment:
session=true would create a session for each slice in parallel, but since here, each slice already has withTrx, doesnt look like we need it... disabled it and all tests are good, including the bulk rest apiece spec